src/components/formik/FormSuccess.module.css

```diff
@@ -0,0 +1,30 @@
+  body {
+        text-align: center;
+        padding: 40px 0;
+        background: #EBF0F5;
+      }
+      p {
+          font-size:20px;
+          margin: 0;
+        }
+      i {
+        color: #9ABC66;
+        font-size: 100px;
+        line-height: 200px;
+        margin-left:-15px;
+      }
+      .card {
```

Comment:
I would prefer if you named `.card` and `.Dcard` to something a little more informative, so someone could take a look at the CSS and understand what these are used for. I think card is still a bit ambiguous (what is the card for?)

Reply:
I should have been more assertive here. I have changed the name `.card` to `.box` as it represents the white box on the submitted page and has also changed `.Dcard` to `.checkmarkCircle` as it represents the circle outside the checkmark sign.